main.js

```diff
@@ -88,28 +68,55 @@ class Timer {
 }
 
 class DOM_Manager {
-    constructor() {
-        this.cardsCountVar = document.getElementsByClassName("card").length;
+    constructor(Fn) {
+        this.callBackFn=Fn;
         this.clickListenerBinded = this.clickListener.bind(this);
+        this.submitOnClickBinded = this.submitOnClick.bind(this);
         this.imgs = [];
-        for(let i=0;i<this.cardsCountVar;i++){
-            this.imgs.push(document.getElementById("img"+i));
-        }
+        this.imagesRandomised=[];
+        document.getElementById("Easy").addEventListener("change", this.difficultOnChange);
+        document.getElementById("Medium").addEventListener("change", this.difficultOnChange);
+        document.getElementById("Hard").addEventListener("change", this.difficultOnChange);
+        document.getElementById("submit").addEventListener("click", this.submitOnClickBinded);
+        this.timer=new Timer();

```

Comment:
Can you explain me why you decided to move `Timer` inside this class?
I don't think `DOMManager` is responsible for working with timer